web/src/engine/websites/CoroCoro.ts

```diff
@@ -25,4 +63,69 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL('/rensai', this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'totalChapterLikes');
+        const { weekdays: { mon, tue, wed, thu, fri, sat, sun } } = this.LocateObjectWithNamedProperty<JSONMangas>(json, 'weekdays');
+        return [mon, tue, wed, thu, fri, sat, sun].reduce((accumulator: Manga[], day) => {
+            const mangas = day.map(manga => new Manga(this, provider, `/title/${manga.id}`, manga.name));
+            accumulator.push(...mangas);
+            return accumulator;
+        }, []);
+
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL(`${manga.Identifier}`, this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'omittedMiddleChapters');
+        const { chapters: { earlyChapters, omittedMiddleChapters, latestChapters } } = this.LocateObjectWithNamedProperty<JSONChapters>(json, 'chapters');
+        return [earlyChapters, omittedMiddleChapters, latestChapters].reduce((accumulator: Chapter[], category) => {
```

Comment:
Are the key names relevant? If not just use generic string keys as recommended for the weekday object.

Reply:
Yes they are because its how chapters are sorted.. 
Website shows them in that order : earlyChapters , omittedMiddleChapters (hidden, its the reason why we do it like that) then latestChapters. If for some reason we omit the key and take them "as they are in the JSON" and for some reason the JSON put omittedMiddleChapters  before earlyChapters  order will be screwed.